.github/workflows/ci.yml

```
@@ -12,25 +12,24 @@ jobs:
         with:
           java-version: 1.8
       - name: Cache Maven packages
+        id: cache
         uses: actions/cache@v2
         with:
-          path: ~/.m2/repository
+          path: |
+            ~/.m2/repository
+            !~/.m2/repository/org/keycloak
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
       - name: Build
-        run: mvn install -B -DskipTests -Pdistribution
-      - name: Tar Maven Repo
-        shell: bash
-        run: tar -czvf maven-repo.tgz -C ~ .m2/repository
-      - name: Persist Maven Repo
-        uses: actions/upload-artifact@v1
-        with:
-          name: maven-repo
-          path: maven-repo.tgz
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: |
+          mvn clean install -B -e -DskipTests -Pquarkus,distribution
+          mvn install -nsu -B -DskipTests -Pdistribution
```

Also to be consistent with Quarkus yaml
```suggestion
          mvn clean install -nsu -B -DskipTests -Pdistribution
```